test/e2e/suites/e2e/clusterctl_upgrade_test.go

```diff
@@ -75,3 +75,26 @@ var _ = Describe("When testing clusterctl upgrades (v0.7=>current) [clusterctl-u
 		}
 	})
 })
+
+var _ = Describe("When testing clusterctl upgrades (v0.8=>current) [clusterctl-upgrade]", func() {
+	ctx := context.TODO()
+	shared.SetEnvVar("DOWNLOAD_E2E_IMAGE", "true", false)
+
+	capi_e2e.ClusterctlUpgradeSpec(ctx, func() capi_e2e.ClusterctlUpgradeSpecInput {
+		return capi_e2e.ClusterctlUpgradeSpecInput{
+			E2EConfig:                       e2eCtx.E2EConfig,
+			ClusterctlConfigPath:            e2eCtx.Environment.ClusterctlConfigPath,
+			BootstrapClusterProxy:           e2eCtx.Environment.BootstrapClusterProxy,
+			ArtifactFolder:                  e2eCtx.Settings.ArtifactFolder,
+			SkipCleanup:                     false,
+			InitWithBinary:                  "https://github.com/kubernetes-sigs/cluster-api/releases/download/" + OldCAPIVersion + "/clusterctl-{OS}-{ARCH}",
+			InitWithProvidersContract:       "v1beta1",
+			InitWithInfrastructureProviders: []string{"openstack:v0.8.0"},
```

Comment:
here too I'm not sure

Reply:
This is ok. Don't change this. This is the 'source' version of an upgrade test, so it can remain at v0.8.0.